aten/src/ATen/native/native_functions.yaml

```
@@ -5867,6 +5867,15 @@
     CPU, CUDA: nansum_out
     MPS: nansum_out_mps
 
+- func: hash_tensor(Tensor self, int[1] dim=[], bool keepdim=False, int mode=0) -> Tensor
+  variants: function, method
+  structured_delegate: hash_tensor.out
+
+- func: hash_tensor.out(Tensor self, int[1] dim=[], bool keepdim=False, int mode=0, *, Tensor(a!) out) -> Tensor(a!)
```

```suggestion
- func: hash_tensor.out(Tensor self, int[1] dim=[], *, bool keepdim=False, int mode=0, *, Tensor(a!) out) -> Tensor(a!)
```

For kwarg-only binding